schema_bump_dry_run_scripts/ontologies/ontology_bump_dry_run.py

```diff
@@ -1,72 +1,131 @@
 #!/usr/bin/env python
-
 import gzip
 import json
 import os
 
 import requests
 
-API_URL = {
-    "prod": "https://api.cellxgene.cziscience.com",
-    "staging": "https://api.cellxgene.staging.single-cell.czi.technology",
-    "dev": "https://api.cellxgene.dev.single-cell.czi.technology",
+BASE_API = {
+    "prod": "api.cellxgene.cziscience.com",
+    "staging": "api.cellxgene.staging.single-cell.czi.technology",
+    "dev": "api.cellxgene.dev.single-cell.czi.technology",
+}
+
+# dataset metadata fields that contain ontology terms
+ONTOLOGY_TYPES = {
+    "assay",
+    "cell_type",
+    "development_stage",
+    "disease",
+    "organism",
+    "self_reported_ethnicity",
+    "sex",
+    "tissue",
 }
 
 
+def get_headers():
+    claims = "openid profile email offline read:collections write:collections"
+    auth0_secrets = json.loads(os.getenv('AUTH0_SECRETS'))
+    response = requests.post(
```

Comment:
This auth flow is blocked on prod to prevent password brute force attacks. You need to generated an API key and use the curation API.

Reply:
What's the best way to do that programmatically? I'm assuming the API Keys eventually expire

Reply:
Looks good